### PR TITLE
Added beginners section

### PR DESCRIPTION
**Pull request by Codedude1**
Here is the live link: https://codedude1.github.io/juniorninjas.github.io/
Screenshot 2021-06-15 at 3 06 01 PM
Still, have to make changes to the course section of the home page.

### CODE REVIEW

**Comment by status404** [comment]
Screenshot from 2021-06-15 20-07-15
TODOs:
1) Remove "Reviews" section
2) Replace "Trainer’s Name ..." with Course duration.
3) Add some cool image for beginners main-image

File: course-details.html [hunk 1]
```diff
@@ -32,8 +32,10 @@
             <div class="row align-items-center">
```
Comment (status404) [inline course-details.html:32]: rename file : course-details.html -> beginners-course.html

File: cource.html [hunk 2]
```diff
@@ -101,10 +101,10 @@ <h2>Special Courses</h2>
                     <div class="single_special_cource">
```
Comment (status404) [inline cource.html:101]: rename file: cource.html -> courses.html

**Comment by Codedude1** [comment]
Only this section remains to be updated! Will it be Okay to add it to the courses page or shall I add it to the home?
Screenshot 2021-06-16 at 12 17 04 AM

**Comment by status404** [comment]
> Only this section remains to be updated! Will it be Okay to add it to the courses page or shall I add it to the home?
> Screenshot 2021-06-16 at 12 17 04 AM

Yes, please add this in courses page, might be as footer.

**Comment by status404** [comment]
Can we make "Course Outline" items as look like a list with numbers or bullets?

**Pull request merged**